src/commands/deploy.ts

```diff
@@ -208,34 +211,47 @@ export async function deployCommand(options: GenezioDeployOptions) {
             });
 
             log.info("Deploying your frontend to the genezio infrastructure...");
-            const url = await deployFrontend(
-                configuration.name,
-                configuration.region,
-                frontend,
-                index,
-                options,
-            ).catch(async (error) => {
-                if (error instanceof Error) {
-                    log.error(error.message);
-                    if (error.message == "No frontend entry in genezio configuration file.") {
-                        exit(0);
+            frontendUrl.push(
+                await deployFrontend(
+                    configuration.name,
+                    configuration.region,
+                    frontend,
+                    index,
+                    options,
+                ).catch(async (error) => {
+                    if (error instanceof Error) {
+                        log.error(error.message);
+                        if (error.message == "No frontend entry in genezio configuration file.") {
+                            exit(0);
+                        }
+                        await GenezioTelemetry.sendEvent({
+                            eventType: TelemetryEventTypes.GENEZIO_FRONTEND_DEPLOY_ERROR,
+                            errorTrace: error.toString(),
+                            commandOptions: JSON.stringify(options),
+                        });
                     }
-                    await GenezioTelemetry.sendEvent({
-                        eventType: TelemetryEventTypes.GENEZIO_FRONTEND_DEPLOY_ERROR,
-                        errorTrace: error.toString(),
-                        commandOptions: JSON.stringify(options),
-                    });
-                }
-                exit(1);
-            });
-            log.info("\x1b[36m%s\x1b[0m", `Frontend successfully deployed at ${url}`);
+                    exit(1);
+                }),
+            );
+            //             log.info("\x1b[36m%s\x1b[0m", `Frontend successfully deployed at ${frontendUrl}`);
 
             await GenezioTelemetry.sendEvent({
                 eventType: TelemetryEventTypes.GENEZIO_FRONTEND_DEPLOY_END,
                 commandOptions: JSON.stringify(options),
             });
         }
     }
+    if (deployClassesResult) {
+        log.info(
+            "\x1b[36m%s\x1b[0m",
```

Comment:
Let's use the `colors` library. The code is much more readable. Right now, reading this, I don't know what color this text will have.

There are multiple places where you shoudh change this.